x/treasury/internal/keeper/keeper.go

```diff
@@ -314,7 +314,7 @@ func (k Keeper) GetTSL(ctx sdk.Context, epoch int64) (res sdk.Int) {
 	bz := store.Get(types.GetTSLKey(epoch))
 
 	if bz == nil {
-		res = sdk.ZeroInt()
+		res = sdk.OneInt()
```

Comment:
Given TSL stands for "Total Staked Luna", it seems to make sense for this GetTSL function to continue to return 0, and have the consumer of this function check for zero value